src/index.tsx

```diff
@@ -11,17 +11,18 @@ import { worker } from 'api/mocks/mock-worker';
 
 import * as serviceWorker from './serviceWorker';
 
-const openReactQueryDevtools = process.env.NODE_ENV === 'development';
+const openReactQueryDevtools = import.meta.env.DEV;
 
-if (process.env.NODE_ENV === 'development') {
+if (import.meta.env.DEV) {
   // eslint-disable-next-line @typescript-eslint/no-var-requires
-  const whyDidYouRender = require('@welldone-software/why-did-you-render');
-  whyDidYouRender(React);
+  // TODO: Uncomment, transform require to import
+  // const whyDidYouRender = require('@welldone-software/why-did-you-render');
+  // whyDidYouRender(React);
```

Comment:
it could be imported dynamically:
```suggestion
  const { default: wdyr } = await import('@welldone-software/why-did-you-render');
  wdyr(React);
```

Reply:
Thanks Barto! I commented this block with TODO and forgot to solve this one